doc/build-osx.md

```diff
@@ -232,28 +219,16 @@ pip3 install ds_store mac_alias
 
 There are many ways to configure Bitcoin Core, here are a few common examples:
 
-##### Wallet (BDB + SQlite) Support, No GUI:
-
-If `berkeley-db@4` is installed, then legacy wallet support will be built.
-If `berkeley-db@4` is not installed, then this will throw an error.
-If `sqlite` is installed, then descriptor wallet support will also be built.
-Additionally, this explicitly disables the GUI.
-
-``` bash
-./autogen.sh
-./configure --with-gui=no
-```
-
-##### Wallet (only SQlite) and GUI Support:
+##### Wallet and GUI Support:
 
-This explicitly enables the GUI and disables legacy wallet support.
+This explicitly enables the GUI.
 If `qt` is not installed, this will throw an error.
 If `sqlite` is installed then descriptor wallet functionality will be built.
 If `sqlite` is not installed, then wallet functionality will be disabled.
 
 ``` bash
 ./autogen.sh
-./configure --without-bdb --with-gui=yes
+./configure --with-gui=yes
```

Comment:
if `qt` is installed, the gui would be built by default, right? So instead of specifying how to `explictly enable gui` maybe its better to specify ho to `explicitly disable gui`? And then add note similar to `sqlite` lines above, saying when gui will be enabled by default? 

My only point is the explicit gui disable step was there before few lines ago, and now its removed.